packages/powerline_fonts.rb

```diff
@@ -21,21 +21,21 @@ class Powerline_fonts < Package
   })
 
   def self.install
-    fonts = "#{CREW_DEST_DIR}/usr/local/share/fonts"
+    fonts = "#{CREW_DEST_PREFIX}/share/fonts"
     FileUtils.mkdir_p(fonts)
 
     Dir.glob('*').each do |f|
       FileUtils.mv(f, fonts) unless ["README.rst", "install.sh"].include? f
     end
 
-    home = "#{CREW_DEST_DIR}/home/chronos/user"
+    home = "#{CREW_DEST_HOME}"
```

Comment:
Why not just use `#{CREW_DEST_HOME}` as the variable? :)

Reply:
That's what I did. I replaced `#{CREW_DEST_DIR}/home/chronos/user` with `#{CREW_DEST_HOME}`.

Reply:
I mean just use `#{CREW_DEST_HOME}` instead of `home` since it's already a variable/constant.

Reply:
Got it.